config/default.schema.json

```diff
@@ -518,6 +518,11 @@
 			"default": "HPI Schul-Cloud",
 			"description": "Look to SC_TITLE, it is a possible short form of it."
 		},
+		"SC_DOMAIN": {
+			"type": "string",
+			"default": "localhost",
```

Comment:
The default values should be what is taken in production. 

Any other default overrides for local setups can be set in ./config/development.json.

Reply:
We have for every instance a value, that's the reason why  we can not set the default for prod.
  We used the default value from the global.js.

Reply:
This is unclear for me. We have on dev, ref and staging setted values that matched to the real locations. 
Also no default is needed on this place. 
If you want to add it for local setups, the default must be defined in ./config/development.json.

Reply:
Ah ok if you look at line 518 this default must also moved. 
We only want default that are used in production as default on this place. Like TIMEOUTS that are the same for all, or the most production instances.

Reply:
ah ok thanks, we will keep that in future.
Done.